src/targets/esp32c3.ts

```
@@ -145,4 +189,26 @@ export class ESP32C3ROM extends ROM {
+    await loader.writeReg(this.RTC_CNTL_WDTWPROTECT_REG, this.RTC_CNTL_WDT_WKEY); // unlock
+    await loader.writeReg(this.RTC_CNTL_WDTCONFIG1_REG, 5000); // set WDT timeout
+    await loader.writeReg(this.RTC_CNTL_WDTCONFIG0_REG, (1 << 31) | (5 << 28) | (1 << 8) | 2); //  enable WDT
+    await loader.writeReg(this.RTC_CNTL_WDTWPROTECT_REG, 0); // lock
+  }
+
+  public async hardReset(loader: ESPLoader) {
+    const isUsingUsbJTAGSerial = await this.useUsbJTAGSerial(loader);
+    if (isUsingUsbJTAGSerial) {
+      await this.rtcWdtReset(loader);
+    } else {
+-      loader.hardReset();
```

ESLint

[error] Replace `-······` with `······-`

(prettier/prettier)